docs/tutorials/accuracy/index.rst

```diff
@@ -0,0 +1,15 @@
+.. _accuracy_scorer_tutorial:
+
+Scorers
+=======
+
+Accuracy Scorers are used for getting the accuracy of the model.
+These tutorials will show you how to use existing DFFML Scorers,
+implement your own scoring algorithm or wrap an existing libraries
+implementation, and package it so others can use it easily.
+
+.. toctree::
+    :maxdepth: 2
+    :caption: Contents:
+
+    accuracy_scorer
```

Comment:
If we follow the same pattern we did in the models, we should probably name this tutorial `mse`, we can wait on adding more tutorials here in other PRs but this is a great start.

Reply:
done.